packages/server/src/integrations/microsoftSqlServer.ts

```diff
@@ -281,8 +281,14 @@ class SqlServerIntegration extends Sql implements DatasourcePlus {
         case MSSQLConfigAuthType.NTLM: {
           const { domain, trustServerCertificate } =
             this.config.ntlmConfig || {}
+
+          if (!domain) {
+            throw Error("Domain must be provided for NTLM config")
+          }
+
           clientCfg.authentication = {
             type: "ntlm",
+            // @ts-ignore
```

Comment:
Could you add a comment to explain why the ts-ignore is necessary? Going forward going to try and follow the `ts-expect-error` practice to note there is a problem that can't be solved here.